File: Source/Bifrost.JavaScript.Specs/utils/for_StringMapping/when_matching_a_string_that_should_match.js
describe("when matching a string that should match", function () {
    var mapping = Bifrost.utils.StringMapping.create({
        format: "{something}/{else}",
        mappedFormat: "whatevva"
    });
    var result = mapping.matches("hello/there");

    it("should match", function () {
        expect(result).toBe(true);
    });
});

File: Source/Bifrost.Specs/Utils/for_StringMapping/when_matching_a_string_that_should_match.cs
using Bifrost.Utils;
using Machine.Specifications;

namespace Bifrost.Specs.Utils.for_StringMapping
{
    public class when_matching_a_string_that_should_match
    {
        static StringMapping mapping = new StringMapping(
                "{something}/{else}",
                "whatevva"
            );
        static bool result;

        Because of = () => result = mapping.Matches("hello/there");

        It should_match = () => result.ShouldBeTrue();
    }
}
